src/js/gi/selectors/calculator.js

```diff
@@ -183,15 +177,18 @@ const getDerivedValues = createSelector(
     } else if (isCorrespondence) {
       tuitionFeesCap = constant.CORRESPONDTFCAP;
     } else if (isPublic && institution.country === 'usa') {
-      tuitionFeesCap = inputs.inState
+      tuitionFeesCap = inputs.inState === 'yes'
                      ? institution.tuitionInState
                      : institution.tuitionOutOfState;
     } else if (isPrivate || isForeign) {
       tuitionFeesCap = constant.TFCAP;
+    } else {
+      // TODO: What should default cap be?
```

Comment:
@saintsoup52 Sure, that constant gets applied for private or foreign school types here, just like the original GIBCT. But how we should handle cases that don't apply to any of the conditions here is what I was wondering. If it's not flight, correspondence, public, private, or foreign, there ends up being an undefined cap. In particular, if it's a for-profit.